src/components/ImageView/Overlay/OverlayComponent.tsx

```diff
@@ -13,16 +13,42 @@ export class OverlayComponentProps {
     overlaySettings: OverlayStore;
     image: ImageItem;
     docked: boolean;
+    top?: number;
+    left?: number;
+    refCanvas?: any;
     onClicked?: (cursorInfo: CursorInfo) => void;
     onZoomed?: (cursorInfo: CursorInfo, delta: number) => void;
+    thisIs?: "left" | "bottom" | "corner" | "inner";
```

Comment:
I think we can remove this because it is always “corner” for type “channel-map-inner”.